evap/staff/forms.py

```diff
@@ -49,10 +72,8 @@ def __init__(self, *args, **kwargs):
         self.fields['course'].choices = choices
 
     def clean(self):
-        if self.cleaned_data['course'] and self.cleaned_data['excel_file']:
-            raise ValidationError('Please select only one of course or Excel file.')
-        if not self.cleaned_data['course'] and not self.cleaned_data['excel_file']:
-            raise ValidationError('Please select either course or Excel File.')
+        if self.cleaned_data['course'] is None:
+            raise ValidationError("Please select a Course from the drop-down.")
```

Comment:
you would probably need to not pass the POST data into the form's constructor in case the other form got submitted. if it's too much of a hassle, leave it as it is.